From a4f12d4849daed23651ab3c23b5ff830aa32b2a0 Mon Sep 17 00:00:00 2001
From: Michal Sekletar <msekleta@redhat.com>
Date: Wed, 3 Feb 2016 10:38:29 +0100
Subject: [PATCH] man: clarify what happens when journalctl is called with
 devpath

Cherry-picked from: 3cea8e06e45fc1757de8f74da29fb5fb181db4eb
Related: #947636
---
 man/journalctl.xml | 10 ++++++++--
 1 file changed, 8 insertions(+), 2 deletions(-)

diff --git a/man/journalctl.xml b/man/journalctl.xml
index 2764f66ed..0981fba72 100644
--- a/man/journalctl.xml
+++ b/man/journalctl.xml
@@ -91,8 +91,14 @@
       paths may be specified. If a file path refers to an executable
       file, this is equivalent to an <literal>_EXE=</literal> match
       for the canonicalized binary path. Similarly, if a path refers
-      to a device node, this is equivalent to a
-      <literal>_KERNEL_DEVICE=</literal> match for the device.</para>
+      to a device node then match is added for the kernel name of the
+      device (<literal>_KERNEL_DEVICE=</literal>). Also, matches for the
+      kernel names of all the parent devices are added automatically.
+      Device node paths are not stable across reboots, therefore match
+      for the current boot id (<literal>_BOOT_ID=</literal>) is
+      always added as well. Note that only the log entries for
+      the existing device nodes maybe queried by providing path to
+      the device node.</para>
 
       <para>Additional constraints may be added using options
       <option>--boot</option>, <option>--unit=</option>, etc, to
